tests/test_auto_vec_detection.py

```diff
@@ -1,10 +1,26 @@
 import pytest
 import numpy as np
 
-from stable_baselines import A2C, ACER, ACKTR, DDPG, DQN, PPO1, PPO2, SAC, TRPO
+from stable_baselines import A2C, ACER, ACKTR, DDPG, DQN, PPO1, PPO2, SAC, TRPO, TD3
 from stable_baselines.common.vec_env import DummyVecEnv
 from stable_baselines.common.identity_env import IdentityEnv, IdentityEnvBox, IdentityEnvMultiBinary, \
     IdentityEnvMultiDiscrete
+from stable_baselines.common.evaluation import evaluate_policy
+
+
+def check_shape(make_env, model_class, shape_1, shape_2):
+    model = model_class(policy="MlpPolicy", env=DummyVecEnv([make_env]))
+
+    env0 = make_env()
+    env1 = DummyVecEnv([make_env])
+
+    for env, expected_shape in [(env0, shape_1), (env1, shape_2)]:
+        def callback(locals_, _globals):
+            assert np.array(locals_['action']).shape == expected_shape
+        evaluate_policy(model, env, n_eval_episodes=5, callback=callback)
+
+    # Free memory
+    del model, env0, env1
```

Comment:
Is this better than `envi.close()`? Also consider making this `try: ... finally:`. (In other projects I've occasionally seen cascading failures from one test failing because other tests then fail due to OOM...)

Reply:
In fact, I think we can remove that. `env.close` does nothing in the case of a `DummyVecEnv` (normally close the viewer)...